index.js

```diff
@@ -1906,7 +1906,7 @@ module.exports = {
                 },
                 'q-applicant-building-and-street-2': {
                     type: 'string',
-                    title: "<span class='govuk-visually-hidden'>Building and street line 2</span>",
```

Comment:
I remember the chat around this change. But will this temporarily be visible to the user, or have we done something to hide it by other means at this point?

Reply:
This has been reverted.